### PR TITLE
feat(oo-ui): add tx hash to data

### PR DESCRIPTION
**Pull request by Gamaranto**
Adds the tx hash as in: https://app.shortcut.com/uma-project/story/4199/within-input-data-add-a-row-before-requestor-with-the-tx-hash-of-the-request-name-request-type-address-data-the-tx-hash-of

Signed-off-by: Gamaranto <francesco@umaproject.org>

### CODE REVIEW

File: src/components/request/Request.tsx [hunk 1]
```diff
@@ -31,8 +30,20 @@ const Request = () => {
...
 
+  console.log(requestTx);
+
+  const { rows, headerCells } = useRequestTableData({
+    chainId,
+    requester: searchParams.get("requester") ?? undefined,
+    identifier: searchParams.get("identifier") ?? undefined,
+    ancillaryData: searchParams.get("ancillaryData") ?? undefined,
+    timestamp: Number(searchParams.get("timestamp")) || undefined,
+    requestTxHash: requestTx,
```
Comment (daywiss) [inline src/components/request/Request.tsx:44]: just fyi for this card: https://app.shortcut.com/uma-project/story/4616/detail-page-needs-to-fill-table-from-request-query-not-query-params, you can accomplish this by falling back to the request properties you will need exported from `useReader`.  I dont know if these are currently exported, but they can be found  by calling `reader.request()` inside the `useOracleReader` function

File: src/components/request/Request.tsx [hunk 2]
```diff
@@ -31,8 +30,20 @@ const Request = () => {
     exploreDisputeTx,
     proposedPrice,
     parsedIdentifier,
+    requestTx,
   } = useReader(state);
 
+  console.log(requestTx);
```
Comment (daywiss) [inline src/components/request/Request.tsx:36]: stray log

**Review by daywiss — APPROVED** [review]
looks good, its up to you if u want to tackle that other card in this PR. id suggest though just doing it in a separate pr because i added another requirement to update the settle page as well.

**Review by Tulun — APPROVED** [review]

File: src/components/request/Request.tsx (hunk 2 above)
Comment (Gamaranto) [reply src/components/request/Request.tsx:36]: thanks removed

**Review by Tulun — APPROVED** [review]
Seems good, just address David's refactor.

**Review by daywiss — APPROVED** [review]
looks good

**Pull request merged**